lib/honeybadger/config/defaults.rb

```diff
@@ -91,6 +91,16 @@ class Boolean; end
         default: 100,
         type: Integer
       },
+      events_batch_size: {
+        description: 'Send events batch if n events have accumulated',
+        default: 100,
+        type: Integer
+      },
+      events_timeout: {
+        description: 'Timeout after which the events batch will be sent regardless (in milliseconds)',
+        default: 30_000,
+        type: Integer
+      },
```

Comment:
What do you think about namespacing these?

```suggestion
      :'events.batch_size' => {
        description: 'Send events batch if n events have accumulated',
        default: 100,
        type: Integer
      },
      :'events.timeout' => {
        description: 'Timeout after which the events batch will be sent regardless (in milliseconds)',
        default: 30_000,
        type: Integer
      },
```

This would allow you do do this in the YAML config:

```yaml
events:
  batch_size: 200
  timeout: 10000
```